### PR TITLE
Update code for error handling

### PR DESCRIPTION
**Pull request by praveena-pappu**

### CODE REVIEW

File: kaggle/src/appsscript.json [hunk 1]
```diff
@@ -12,14 +12,10 @@
     "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title\u003dKaggle%20Connector%20Issue",
     "termsOfServiceUrl": "https://policies.google.com/terms",
     "privacyPolicyUrl": "https://policies.google.com/privacy",
-    "description": "Connect to and visualize Kaggle datasets. This connector supports a single CSV file up to 20MB size for each data source. Add the connector multiple times for multiple data sources. Steps to get your Kaggle API key: https://www.kaggle.com/docs/api#getting-started-installation-\u0026-authentication",
+    "description": "Connect to and visualize Kaggle datasets. This connector supports a single CSV file up to 20MB size for each data source. Add the connector multiple times for multiple data sources. Steps to get your Kaggle API key: https://www.kaggle.com/docs/api#getting-started-installation-\u0026-authentication. Use the deployment link: https://datastudio.google.com/u/0/datasources/create?connectorId=AKfycbz8N8nRisgB6Xi7Z-HP70vuWfz4nQ378CQaOwE_tTs1Sn6Avq_AiDaB0DqqfXFxIaHxpA to test the functionality of the connector.",
```
Comment (diminishedprime) [inline kaggle/src/appsscript.json]: I don't think we want to include the deployment link in the description.

File: kaggle/src/connector.js [hunk 2]
```diff
@@ -78,20 +73,20 @@ function getSchema(request) {
...
-  }
-
-  var fileIsSmall = isFileSmall(config);
-  if (fileIsSmall === false) {
-    throwConnectorError('Please use smaller than 20MB csv files.', true);
+    throwConnectorError('Only .csv filetypes are supported');
+  } else if (fileTypeIsSupported === true) {
+    var fileIsSmall = isFileSmall(config);
+    if (fileIsSmall === false) {
+      throwConnectorError('Please use .csv files less than 20MB in size.');
```
Comment (diminishedprime) [inline kaggle/src/connector.js]:
```suggestion
      throwConnectorError('Please use .csv files smaller than 20MB.');
```

Comment (praveena-pappu) [reply kaggle/src/connector.js]: Updated the changes requested

File: google-fit/src/appsscript.json [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
   "dataStudio": {
     "name": "Google Fit",
-    "company": "Google Data Studio Developer Relations",
-    "company_url": "https://developers.google.com/datastudio/",
     "logoUrl": "https://gstatic.com/images/branding/product/1x/gfit_512dp.png",
+    "company": "Google Data Studio Developer Relations",
+    "companyUrl": "https://developers.google.com/datastudio/",
     "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/google-fit/README.md",
-    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Google%20Fit%20Connector%20Issue",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title\u003dGoogle%20Fit%20Connector%20Issue",
```
Comment (diminishedprime) [inline google-fit/src/appsscript.json]: please change this link back so it works correctly.

File: kaggle/src/connector.js (hunk 2 above)
Comment (diminishedprime) [reply kaggle/src/connector.js]: please fix.

**Pull request merged**